RFC.md

```diff
@@ -0,0 +1,63 @@
+# RFC
+The ISIS project uses a Request for Comment (RFC) model whereby major potential changes to the code base, data area, or binary delivery process are proposed, iterated on by any interested parties, and potentially adopted. Right now, RFCs are being housed n this repository's [discussion board](https://github.com/USGS-Astrogeology/ISIS3/discussions?discussions_q=label%3ARFC).
+
+We encourage all contributors and users to review open RFCs and comment as these proposed changes will impact use of the software. 
+
+# RFC policies
+
+RFC adoption is driven by general consensus in the RFC comments.
+
+# RFC lifecycle
+
+1. RFC is written and posted on the discussion board
+2. RFC is advertised and discussion proceeds until consensus is reached on what changes if any are needed
```

Comment:
RFC is advertised and discussion proceeds until consensus is reached (the rest of the sentence does not make sense)